src/bitstream.rs

```diff
@@ -49,6 +49,17 @@ impl<'a> Bitstream<'a> {
         Ok(self.n & 1)
     }
 
+    pub fn read_byte(&mut self) -> Option<u8> {
+        if self.buffer.is_empty() {
+            return None;
+        }
+        let byte = self.buffer[0];
+        self.buffer = &self.buffer[1..];
+        self.remaining = 0;
+        self.n = 0;
+        Some(byte)
+    }
```

Comment:
Shouldn't this behave more like `read_bits_oneword(8)`?

Reply:
@Lonami I think no. I tried to do this and it doesn't work.